smaug/python/tensor_utils.py

```diff
@@ -66,3 +66,22 @@ def get_tensor_data_op(tensor):
       data_op_output.source = (node, 0)
       return data_op_output
   return None
+
+def get_tensor_reorder_op(tensor, layout):
+  """Return the output of a reorder op if this tensor already has one.
+
+  Args:
+    tensor: A Tensor.
+    layout: The target layout.
+
+  Returns:
+    If found, the output tensor of the found reorder op. Otherwise, None is
+    returned.
+  """
+  for node in tensor.targets:
+    if node.op == types_pb2.Reorder and node.output_tensors[
+        0].shape.layout == layout:
+      reorder_op_output = from_tensor_proto(node.output_tensors[0])
```

Comment:
Again - why not just return the output of the operation directly? Why is it necessary to create a new tensor?

Reply:
As stated in the previous reply, because the output of the operation is a `TensorProto`, but we want a `Tensor`.